leasing/report/lease/decision_conditions_report.py

```diff
@@ -128,10 +130,14 @@ def get_data(self, input_data):
         if input_data["service_unit"]:
             qs = qs.filter(decision__lease__service_unit__in=input_data["service_unit"])
 
-        if input_data["supervision_exists"] is not None:
-            qs = qs.filter(
-                supervision_date__isnull=not input_data["supervision_exists"]
+        if (
```

Comment:
This gets pretty complicated to understand what is going on, is there any way to simplify it?
How about
```if(...is None and == "False"):
    qs = ...filter(..._isnull=True)
```

There's many solutions to simplify it

Reply:
There is a distinction between null and False. If it's null, no filter is applied and if  it's False then only show those that don't have supervision_exists set. However, I agree that this can be simplified. I'll turn it into a simpler if else statement.